setup.py

```diff
@@ -152,7 +152,7 @@ def build_rdkit(self, ext):
         # Start build process
         os.chdir(str("rdkit"))
 
-        if rdkit_tag == "Release_2023_09_6":
+        if rdkit_tag = "Release_2024_03_2"
```

Comment:
@kuelumbus it seems that the github action introduced a typo here (removed an `=` and the `:` at the end which prevents the build from working

Reply:
Thanks for finding this. I think if have this fixed in the main branch.

Reply:
Oh I didn't see that part :( 
Hope they'll resolve it soon.

Thanks for the reply.